src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefLocalTests.cs

```diff
@@ -1600,5 +1598,33 @@ static void M()
                 Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion6, "ref").WithArguments("byref locals and returns", "7").WithLocation(6, 22)
                 );
         }
+
+        [Fact]
+        public void RefVarSemanticModel()
+        {
+            var text = @"
+class Program
+{
+    static void M()
+    {
+        int i = 0;
+        ref var x = ref i;
```

Comment:
Consider also testing GetAliasInfo API.

---
In reply to: [95439359](https://github.com/dotnet/roslyn/pull/16342#discussion_r95439359) [](ancestors = 95439359)